shared/chat/conversation/attachment-popup/index.native.js

```diff
@@ -32,15 +32,10 @@ const AttachmentPopup = ({message, detailsPopupShowing, isZoomed, onCloseDetails
 
   if (!previewType || previewType === 'Other') {
     return (
-      <PopupDialog onClose={onClose} fill={true} styleContainer={{
+      <Box style={{
         ...globalStyles.flexBoxColumn,
+        ...globalStyles.fillAbsolute,
         backgroundColor: globalColors.white,
-        borderRadius: 0,
-        bottom: 0,
-        left: 0,
-        position: 'absolute',
-        right: 0,
-        top: 20,
```

Comment:
yeah this was temp until we allowed full screen, else the time would bleed on top of the component and look bad